ionoscloud/resource_apigateway_test.go

```diff
@@ -95,8 +83,8 @@ func TestAccAPIGateway_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("ionoscloud_apigateway.example", "name", "example"),
 					resource.TestCheckResourceAttr("ionoscloud_apigateway.example", "logs", "false"),
 					resource.TestCheckResourceAttr("ionoscloud_apigateway.example", "metrics", "true"),
-					resource.TestCheckResourceAttr("ionoscloud_apigateway.example", "custom_domains.0.name", "example.com"),
-					resource.TestCheckResourceAttr("ionoscloud_apigateway.example", "custom_domains.0.certificate_id", "00000000-0000-0000-0000-000000000000"),
+					// can't be used in tests, as it requires a reachable domain name
+					// resource.TestCheckResourceAttr("ionoscloud_apigateway.example", "custom_domains.0.name", "example.com"),
```

Comment:
why we don't remove that line instead of commenting?

Reply:
we might want to add it back if we create an externally available name. we can remove it